climakitae/data_export.py

```diff
@@ -44,8 +44,12 @@ def export_to_csv(data_to_export,save_name,**kwargs):
     csv_nrows = len(to_save.index)        
     if (csv_nrows > excel_row_limit):
         print("WARNING: Dataset exceeds Excel limit of "+
-                      str(excel_row_limit)+" rows.")        
-    print("Compressing data... This can take a bit...")
+                      str(excel_row_limit)+" rows.") 
+        
+    
+    metadata_to_file(data_to_export,save_name)
+    print("Converting and compressing data... This can take a bit...")
+    
```

Comment:
This as well. If the download is fast, it's confusing to notify the user to expect a long wait time